### PR TITLE
ARTEMIS-2359 Upgrade to Guava 24.1

### PR DESCRIPTION
**Pull request by brusdev**
CVE-2018-10237 guava: Unbounded memory allocation in AtomicDoubleArray
and CompoundOrdering classes allow remote attackers to cause a denial
of service.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -800,6 +800,12 @@
             <scope>provided</scope>
             <!-- License: Apache 2.0 -->
          </dependency>
+         <dependency>
```
Comment (clebertsuconic) [inline pom.xml]: Guava is only used by airlift.. Do we really need to add the dependency globally?

**Comment by clebertsuconic** [comment]
can you rebase please?

File: pom.xml (hunk 1 above)
Comment (brusdev) [reply pom.xml]: I see your point but airlift/airline is added as global dependency and Guava is used by artemis-cli (XmlDataExporterUtil), by integrations-tests (MethodCalledVerifier) by airlift/airline and by hawtio/web.
Airlift/airline is added as global dependency, as artemis-cli dependency and as artemis-distribution dependency.
Hawtio/web is added as artemis-console dependency.

**Comment by brusdev** [comment]
rabased

**Pull request merged**